### ISSUE TITLE
deletion in case of internet cut for X hours

### ISSUE THREAD

**Issue by xyz-nobody** [x13a/Wasted#24]
Is it possible to delete the phone if it can not connect to the internet for more than X hours / days?

Ex: you can leave your phone without activity for 2 days.

But if we have no internet connection (ping or other) it is that there is a problem.

The ideal would be to be able to customize a URL where we can hide a small text file and if inside there is no Wasted code, it erases the phone after X hours.

**Comment by x13a** [comment]
For now I have no plans on this. Internet can be lost for many reasons. I
will trigger unexpected wipes.

**Comment by xyz-nobody** [comment]
it's true that internet can be lost for many different reasons, but it's a real inactivity :)

If I travel I know I have to increase the time.
Otherwise if I don't have internet for X hours or Days I know there is a problem.

My phone has been stolen, an attacker has my phone.

I work 90% of the time in a specific user profile. But I have several profiles that I rarely use. 

So even if I delete my user profile with duress or if my phone is taken by an attacker when I'm not in my sensitive profile it doesn't help me much. Because from one profile I have no solution to delete all the phone.

An attacker will always try to shut down the internet and turn off the phone so he can try to defeat the security or force the user to talk and give his code.

Because attackers know that a code can be sent remotely to wipe the phone, that's why their first reflex is to put the phone in airplane mode (by the way I love your fake airplane mode) or to try to turn off the phone or the network by putting it in a suitcase expres where it cuts all radio communication (4g, wifi etc).

**Comment by x13a** [comment]
I think to move "wipe on inactivity" to companion app and implement it there.

**Comment by Beach77** [comment]
Yeah I love this idea very much. If someone who knows what they are doing takes your phone they will put it in a bag/case that blocks all connections to the internet so that you can't remotely wipe it. With this idea they would be in for a suprise.
